modules/nw-bpfman-operator-deploy.adoc

```diff
@@ -0,0 +1,107 @@
+// Module included in the following assemblies:
+//
+// * networking/network_security/bpfman-operator/bpfman-operator-deploy.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-bpfman-operator-deploy_{context}"]
+= Deploying a containerized eBPF program
+
+As a cluster administrator, you can deploy an ePBF program to nodes on your cluster. In this procedure, a sample containerized eBPF program is installed in the `go-xdp-counter` namespace.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+* You have an account with administrator privileges.
+* You have installed the bpfman Operator.
+
+.Procedure
+
+. To download the manifest, enter the following command:
++
+[source,terminal]
+----
+$ curl https://github.com/bpfman/bpfman/releases/download/v0.5.0/go-xdp-counter-install-selinux.yaml -o go-xdp-counter-install-selinux.yaml
```

Comment:
Test the curl command. I tried it and it created an empty file. I replaced `curl` with `wget` (removed the -o as well) and downloaded fine, so the link is valid. Not sure why the curl failed for me.

Reply:
I found the issue. The link is a redirect under the covers. wget follows redirects by default, curl does not. To fix the command, add a -L:
```suggestion
$ curl -L https://github.com/bpfman/bpfman/releases/download/v0.5.0/go-xdp-counter-install-selinux.yaml -o go-xdp-counter-install-selinux.yaml
```